### ISSUE TITLE
Automatically support most versions of MC that forge does.

### ISSUE THREAD

**Issue by wgaylord** [phase/flavor#1]
You can make it automatically down load fernflower patches and mcp mappings from
Minecraft Forge. 

http://files.minecraftforge.net/maven/de/oceanlabs/mcp/mcp/1.9/mcp-1.9-srg.zip

That would get you the 1.9 patches and mappings.

So you could dynamically get what ever you need for the version you are targeting.

**Comment by wgaylord** [comment]
Any comment?

**Comment by phase** [comment]
Grabbing the files from there shouldn't be too hard. Their patches don't seem to match up with mine, probably because I'm using the wrong FernFlower.

```
http://files.minecraftforge.net/maven/de/oceanlabs/mcp/mcp/1.9.4/mcp-1.9.4-srg.zip
http://files.minecraftforge.net/maven/de/oceanlabs/mcp/mcp/{mcversion}/mcp-{mcversion}-srg.zip
```

**Comment by wgaylord** [comment]
Yeah there patches use forge version of fernflower which has been beefed up a lot.

**Comment by PizzaCrust** [comment]
and the python scripts to fix the common fernflower mistakes

**Comment by wgaylord** [comment]
That too they have one script named clean up the cleans up the source a bit in MCP.

**Comment by PizzaCrust** [comment]
Really? I didn't know that.
